### PR TITLE
[10.0][ADD] stock_exclude_to_remove_lot

### PR DESCRIPTION
**Pull request by rousseldenis**
Tried with this to be not so much invasive (e.g. modifying views)

Note: If you want to scrap expired lots, look at stock-logistics-warehouse/stock_available_product_expiry module.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can we please avoid plurals in module name?

File: stock_exclude_to_remove_lot/models/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
```
Comment (Cedric-Pigeon) [inline stock_exclude_to_remove_lot/models/stock_production_lot.py]: 2019...

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

File: stock_exclude_to_remove_lot/models/stock_production_lot.py (hunk 1 above)
Comment (rousseldenis) [reply stock_exclude_to_remove_lot/models/stock_production_lot.py]: @Cedric-Pigeon done

**Review by Cedric-Pigeon — APPROVED** [review]

File: stock_exclude_to_remove_lot/views/stock_picking_type.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="view_picking_type_form">
```
Comment (tbaden) [inline stock_exclude_to_remove_lot/views/stock_picking_type.xml:7]: just aesthetic, but id before model would be nice.

**Review by tbaden — APPROVED** [review]

**Comment by rousseldenis** [comment]
@OCA/logistics-maintainers

**Pull request merged**